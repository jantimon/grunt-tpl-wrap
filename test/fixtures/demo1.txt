Demo 1 Text File
