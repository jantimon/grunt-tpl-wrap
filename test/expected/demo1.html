<h1> Wrapped test/fixtures/demo1.txt </h1>
<p>
Demo 1 Text File

</p>